src/context/viewer-context.tsx

```diff
@@ -189,10 +205,11 @@ function useAuthedViewer() {
       setSession: auth.setSession, // TODO: This isn't exported in auth, so isn't available here.
       handleAccessTokenAuthentication: handleAccessTokenAuthentication,
       isAuthenticated: () => auth.isAuthenticated(),
-      authToken: auth.getAuthToken(),
+      authToken: authToken,
       requestSignInEmail: (email: any) => auth.requestSignInEmail(email),
       loading,
       refreshUser: auth.refreshUser,
+      ability,
     }),
     [viewer, loading],
```

Comment:
you'll likely want to add the authToken to the deps array so that it will update the memoized value when/if it changes for various reasons